src/scenes/Root/Sidebar/Sidebar.tsx

```diff
@@ -0,0 +1,158 @@
+import {
+  Button,
+  List,
+  ListItem,
+  ListItemText,
+  makeStyles,
+  Menu,
+  MenuItem,
+  Typography,
+} from '@material-ui/core';
+import { Add, FolderOpen, Language, People } from '@material-ui/icons';
+import { FC, useState } from 'react';
+import * as React from 'react';
+import { CordIcon, InternshipsIcon } from '../../../components/Icons';
+import { SidebarListLink } from './SidebarListLink';
+import { SwooshIcon } from './SwooshIcon';
+
+const useStyles = makeStyles(({ spacing, palette }) => ({
+  listContent: {
+    padding: spacing(4, 2),
+  },
+  list: {
+    backgroundColor: '#3c444e',
+    maxWidth: '248px',
+    height: '100%',
+    padding: 0,
+  },
+  createListItem: {
+    justifyContent: 'center',
+    marginBottom: spacing(3),
+  },
+  createButton: {
+    width: '184px',
+    backgroundColor: '#ff5a5f',
+  },
+  menuHeaderListItem: {
+    padding: 0,
+    marginBottom: spacing(1),
+  },
+  menuHeaderText: {
+    color: '#d1dadf',
+    marginLeft: spacing(2),
+  },
+  menuText: {
+    color: palette.text.secondary,
+  },
+  links: {
+    padding: spacing(0, 2),
+  },
+  floating: {
+    position: 'absolute',
+    top: spacing(3),
+    left: spacing(4),
+  },
+  cordIcon: {
+    height: '45px',
+    width: '38px',
+    color: palette.primary.contrastText,
+    marginBottom: spacing(2),
+  },
+  copyright: {
+    color: '#fbfbfb',
+    fontWeight: 300,
+  },
+}));
+
+export const Sidebar: FC = () => {
+  const {
+    list,
+    listContent,
+    createListItem,
+    createButton,
+    menuHeaderText,
+    menuHeaderListItem,
+    floating,
+    cordIcon,
+    copyright,
+  } = useStyles();
+  const [anchorEl, setAnchorEl] = useState(null);
+
+  const handleClick = (e: any) => {
+    setAnchorEl(e.currentTarget);
+  };
+
+  const handleClose = () => setAnchorEl(null);
+
+  return (
+    <List className={list} component="nav" aria-label="sidebar">
```

Comment:
I don't think it makes sense for the entire sidebar to be a list. Just the actual list section.